docs/guides/how_to_guides/configuring_data_contexts/how_to_instantiate_a_data_context_on_an_databricks_spark_cluster.rst

```diff
@@ -0,0 +1,198 @@
+.. _how_to_instantiate_a_data_context_on_a_databricks_spark_cluster:
+
+How to instantiate a Data Context on Databricks Spark cluster
+=========================================================
+
+This guide will help you instantiate a Data Context on an Databricks Spark cluster.
+
+The guide demonstrates the recommended path for instantiating a Data Context without a full configuration directory and without using the Great Expectations :ref:`command line interface (CLI) <command_line>`.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+    - :ref:`Followed the Getting Started tutorial and have a basic familiarity with the Great Expectations configuration<getting_started>`.
```

Comment:
Also assume you have a Databricks Sparks notebook available?

Reply:
great point. will add this now